### PR TITLE
feat: ✅ integrating crowdin

### PR DESCRIPTION
**Pull request by appsplash99**
contains supported languages + crowdin pipeline integration

### CODE REVIEW

File: crowdin.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+files:
+  - source: /public/locales/en/*.json
+    translation: /public/locales/%two_letters_code%/%original_file_name%
```
Comment (abhagsain) [inline crowdin.yml:3]: Is this `%` symbol intentional?

File: pages/language/index.tsx [hunk 2]
```diff
@@ -1,27 +1,104 @@
 import React from "react";
-import { Container, Flex } from "@chakra-ui/react";
-import LanguageIcons from "components/sections/language/LanguageIcons";
+import { Button, VStack, Text, Image } from "@chakra-ui/react";
+import { ArrowForwardIcon } from "@chakra-ui/icons";
```
Comment (abhagsain) [inline pages/language/index.tsx:3]: Don't we have a custom icon for this? I think there's an Icon for (Right Icon) `->` in the `Icons` file.

**Review by abhagsain — APPROVED** [review]
@appsplash99 There are two comments please check them. But approving the PR. Resolve them and let's merge it.

File: pages/language/index.tsx (hunk 2 above)
Comment (appsplash99) [reply pages/language/index.tsx]: resolved 👍

**Pull request merged**